### PR TITLE
Added Tangent Function

### PR DESCRIPTION
**Pull request by AggRag**

### CODE REVIEW

File: Calculator.py [hunk 1]
```diff
@@ -17,3 +19,14 @@ def average(self, nums):
     nums_sum = self.sum(nums)
     result = nums_sum / l
     return result
+  def tangent(self,nums):
+    result = []
+    for i in nums:
+        result.append(math.tan(i))
+    return result
```
Comment (itaditya) [inline Calculator.py]: use Python3 map to generate a list of sines from list of angles. Take a look at this PR https://github.com/itaditya/python-calculator/blob/master/Calculator.py#L22

You won't need lambda though

File: Calculator.py [hunk 2]
```diff
@@ -17,3 +19,14 @@ def average(self, nums):
...
+ def cosine(self,nums):
+    result = []
+    for i in nums:
+        result.append(math.cos(i))
+    return result
```
Comment (itaditya) [inline Calculator.py:25]: make a separate PR for this and use map

**Comment by itaditya** [comment]
fix the merge conflicts

File: cli.py [hunk 3]
```diff
@@ -19,6 +19,10 @@ def calculate(operation, integers):
     return calc.multiply(integers)
   if (operation == 'average'):
     return calc.average(integers)
+  if (operation == 'tangent'):
```
Comment (itaditya) [inline cli.py]:
```suggestion
  if (operation == 'tan'):
```

Please look at the issue description. There I have said that usage would be

```sh
python3 cli.py tan 30
```

File: cli.py [hunk 4]
```diff
@@ -19,6 +19,10 @@ def calculate(operation, integers):
...
+    return calc.tangent(integers) 
+  if (operation == 'cosine'):
```
Comment (itaditya) [inline cli.py]:
```suggestion
  if (operation == 'cos'):
```

File: Calculator.py (hunk 1 above)
Comment (AggRag) [reply Calculator.py]: Done

File: Calculator.py (hunk 2 above)
Comment (AggRag) [reply Calculator.py:25]: Done

File: cli.py (hunk 3 above)
Comment (AggRag) [reply cli.py]: Done

File: cli.py (hunk 4 above)
Comment (AggRag) [reply cli.py]: Done
